### PR TITLE
add vt_graph export module

### PR DESCRIPTION
**Pull request by alvarogf-vt**

### CODE REVIEW

File: misp_modules/lib/vt_graph_parser/importers/base.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+"""vt_graph_parser.importers.base.
+
+This module provides a common method to import graph from misp attributes.
+"""
+
+
+import vt_graph_api
```
Comment (chrisr3d) [inline misp_modules/lib/vt_graph_parser/importers/base.py]: This one needs to be added in the pipenv config files and eventually in the requirements file, but no big deal we can do that after the PR is merged

File: misp_modules/lib/vt_graph_parser/importers/base.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+from lib.vt_graph_parser.helpers.rules import MispEventRule
```
[the remark this answers is not in the document]
Comment (alvarogf-vt) [reply misp_modules/lib/vt_graph_parser/importers/base.py]: that's right

File: misp_modules/lib/vt_graph_parser/importers/base.py (hunk 1 above)
Comment (alvarogf-vt) [reply misp_modules/lib/vt_graph_parser/importers/base.py]: I will add it to those files.

**Comment by chrisr3d** [comment]
Thank you very much!

**Pull request merged**